src/parser/parser.cpp

```diff
@@ -166,8 +166,56 @@ bool Parser::WriteParseTree(const std::string &filepath) {
     ofs << parse_tree_.dump(INDENTATION) << '\n';
     return true;
   }
-
   return false;
 }
 
+/**
+ * Supportive function for Parser::FormattedJSON
+ * @param value
+ * @return { text: { name: "value" } }
+ */
+json GetTextNode(const std::string &value) {
```

Comment:
Make utility functions private members?
Just a suggestion

Reply:
I think for a single case it does need to be a member at all!!

Reply:
okay, sounds good